evap/staff/templates/staff_user_merge.html

```diff
@@ -10,7 +10,12 @@
 
 {% block content %}
     {{ block.super }}
-    <h3>{% trans 'Merge users' %}</h3>
+    <div class="d-flex">
+        <h3>{% trans 'Merge users' %} </h3> 
+        <div class="ms-auto"> 
+            <a class="btn btn-sm btn-light" href="{% url 'staff:user_merge' main_user_id=other_user.pk other_user_id=main_user.pk %}">{% trans 'Swap main user' %}  </a>
+        </div>
```

Comment:
Nit: There is some extra whitespace here
```suggestion
        <h3>{% trans 'Merge users' %}</h3> 
        <div class="ms-auto"> 
            <a class="btn btn-sm btn-light" href="{% url 'staff:user_merge' main_user_id=other_user.pk other_user_id=main_user.pk %}">{% trans 'Swap main user' %}</a>
        </div>
```